modules/infra/src/main/scala/scaladex/infra/SqlDatabase.scala

```diff
@@ -165,8 +165,11 @@ class SqlDatabase(conf: DatabaseConfig, xa: doobie.Transactor[IO]) extends Sched
   override def updateProjectCreationDate(ref: Project.Reference, creationDate: Instant): Future[Unit] =
     run(ProjectTable.updateCreationDate.run((creationDate, ref))).map(_ => ())
 
-  private def run(update: doobie.Update0): Future[Unit] =
-    update.run.transact(xa).unsafeToFuture().map(_ => ())
+  override def getAllGroupIds(): Future[Seq[Artifact.GroupId]] =
+    run(ArtifactTable.selectGroupIds.to[List])
```

Comment:
Why `List` rather than `Seq`
```suggestion
    run(ArtifactTable.selectGroupIds.to[Seq])
```

Reply:
Ok.. I thought it wasn't possible, so I was always doing `to[List]`